### ISSUE TITLE
build.ps1 fails in non-English culture because of the culture sensitive test

### ISSUE THREAD

**Issue by SIkebe** [cake-build/cake#2541]
### What You Are Seeing?
`.\build.ps1` in this repo fails to execute.

### What is Expected?
`.\build.ps1` executes without error.

### What version of Cake are you using?
Latest

### Are you running on a 32 or 64 bit system?
64bit

### What environment are you running on?  Windows? Linux? Mac?
Windows

### Are you running on a CI Server?  If so, which one?
No

### How Did You Get This To Happen? (Steps to Reproduce)
Clone this repo and execute `.\build.ps1` under the repository root.
Test fails here.

https://github.com/cake-build/cake/blob/90ba67fa94d2650fa1a7b445a6515c29084b707b/src/Cake.Common.Tests/Unit/EnvironmentAliasesTests.cs#L161-L165

### Output Log
```
PS C:\GitHub\cake> .\build.ps1
.
.
.
[xUnit.net 00:00:03.6363220]     Cake.Common.Tests.Unit.EnvironmentAliasesTests+TheEnvironmentVariableMethod.Should_Throw_If_Value_Does_Not_Convert [FAIL]
失敗   Cake.Common.Tests.Unit.EnvironmentAliasesTests+TheEnvironmentVariableMethod.Should_Throw_If_Value_Does_Not_Convert
エラー メッセージ:
 Assert.Contains() Failure
Not found: not a valid value
In value:  abc は Int32 の有効な値ではありません。
スタック トレース:
   場所 Cake.Common.Tests.Unit.EnvironmentAliasesTests.TheEnvironmentVariableMethod.Should_Throw_If_Value_Does_Not_Convert() 場所 C:\GitHub\cake\src\Cake.Common.Tests\Unit\EnvironmentAliasesTests.cs:行 165
Results File: C:/GitHub/cake/artifacts/v0.34.0-alpha0012/test-results/Cake.Common.Tests_full_TestResults_2019-04-29_15-30-42-489.xml

テストの合計数: 3897。成功: 3896。失敗:1。スキップ: 0。
テストの実行に失敗しました。
テスト実行時間: 5.1722 秒
An error occurred when executing task 'Run-Unit-Tests'.

----------------------------------------
Teardown
----------------------------------------
Starting Teardown...
Finished running tasks.
Error: One or more errors occurred. (.NET Core CLI: Process returned an error (exit code 1).)
        .NET Core CLI: Process returned an error (exit code 1).
```

**Comment by drakegens** [comment]
I might take a stab at this one. Any context as I am first time contributor to this project would be appreciated! Including if this would be a good first issue to tackle.

**Comment by devlead** [comment]
Fixed by #2651

**Comment by SIkebe** [comment]
Thanks!
